.gitlab-ci.yml

```diff
@@ -24,7 +24,7 @@ build:
     name: "$PROJECT_NAME"
     expire_in: 1 hour
     paths:
-    - target/*
+    - toolchain-fitnesse-plugin/target/*
```

Comment:
volgens mij is de gitlab ci file overbodig; we hebben de boel nu in github staan, niet meer in gitlab. Deze file kan helemaal weg.